src/sage/combinat/free_module.py

```diff
@@ -498,15 +498,23 @@ def change_ring(self, R):
             Free module generated by {'a', 'b', 'c'} over Rational Field
             sage: F_QQ.change_ring(ZZ) == F
             True
+
+            sage: T = F.tensor(F); T
+            Free module generated by {'a', 'b', 'c'} over Integer Ring # Free module generated by {'a', 'b', 'c'} over Integer Ring
+            sage: T.change_ring(QQ)
+            Free module generated by {'a', 'b', 'c'} over Rational Field # Free module generated by {'a', 'b', 'c'} over Rational Field
```

Comment:
```suggestion
            sage: T = F.tensor(F); T
            Free module generated by {'a', 'b', 'c'} over Integer Ring 
             # Free module generated by {'a', 'b', 'c'} over Integer Ring
            sage: T.change_ring(QQ)
            Free module generated by {'a', 'b', 'c'} over Rational Field # Free module generated by {'a', 'b', 'c'} over Rational Field
```

best to wrap long output so it can be read in the formatted doc without horizontal scrolling - https://doc.sagemath.org/html/en/developer/coding_basics.html#writing-testable-examples